Development-Guide/Git-Guides-and-Standards/Commit-Messages.adoc

```diff
@@ -19,28 +19,33 @@ Further paragraphs come after blank lines.
 - Bullet points are okay, too.
 - Typically a hyphen or asterisk is used for the bullet, followed by a
   single space. Use a hanging indent.
-```
-
-Your commit title should be phrased so it can answer the question "If this commit is applied, it will..."
-
-### Example commit message
 
-```
-Change the units of cell voltages from mV to V
-
-Based on testing and feedback....
+Fixes/Continues <Your-JIRA-Ticket>
 ```
 
 #### Subject line
 
-The subject line should be able to describe the effects of the patch:
+The subject line should be able to answer the question:
 
 _If applied, this commit will <your subject line here>_
 
 #### Information in commit messages
 
-The summary should be in paragraph form and should discuss the _whys_ of the change,
-linking to any documentation or references depending on how complicated the change is.
+The summary should be in paragraph form and should discuss the _whys_ of the change, for example, why the specific bug occured or why this peculiar change works.
+
+Documentation or references can be linked depending on how complicated the change is.
+
+If the developer & reviewer agree that the change is trivial or simple enough that just subject line is explanatory, then an extra paragraph is not needed.
```

Comment:
that just subject line
**Really** nitpicky but assuming *the* is meant in the middle